### PR TITLE
send button messages for the start and stop actions added in this branch

### PR DESCRIPTION
**Pull request by itoogii**
I have updated the button send messages for play and record buttons in popup window, and then updated the content script for the new message inputs. The additional changes will be implemented in separate branch and PR. This PR is only limited to this small change.

### CODE REVIEW

File: src/pages/Content/index.js [hunk 1]
```diff
@@ -10,10 +10,18 @@ chrome.runtime.onMessage.addListener(function (msgObj) {
       console.log('Should start recording here');
       sendMessage({ action: 'test', value: 'Hi from content (recorder)!' });
       break;
+    case 'stopRecording':
+      console.log('Should stop recording here');
+      sendMessage({ action: 'test', value: 'Hi from content (recorder)!' });
```
Comment (agamm) [inline src/pages/Content/index.js:15]: The action and value are not consistent with the stoprecording event, no?

Comment (agamm) [reply src/pages/Content/index.js:15]: and here we will need to call some kind of function to stop the recording and send back the Popup all the events we found.

File: src/pages/Content/index.js [hunk 2]
```diff
@@ -10,10 +10,18 @@ chrome.runtime.onMessage.addListener(function (msgObj) {
...
+      break;
     case 'startPlaying':
       console.log('Should start playing recording...');
       sendMessage({ action: 'test', value: 'Hi from content (player)!' });
       break;
+    case 'stopPlaying':
+      console.log('Should stop playing recording...');
+      sendMessage({ action: 'test', value: 'Hi from content (player)!' });
```
Comment (agamm) [inline src/pages/Content/index.js:23]: Again, the message values and action are not consistent with this event.
Plus here we need to send back Popup an indication if the play finished successfully or not.

**Review by agamm — CHANGES_REQUESTED** [review]
A few minor changes needed.

File: src/pages/Content/index.js (hunk 1 above)
Comment (itoogii) [reply src/pages/Content/index.js:15]: This PR is for updating the button click sendMessages, and the content script sendMessage is there to keep the same content of previous cases. I think we should update the sendMessage or respective actions for the Content script in the separate PR.

File: src/pages/Content/index.js (hunk 2 above)
Comment (itoogii) [reply src/pages/Content/index.js:23]: Same as the other

**Review by agamm — APPROVED** [review]
If you are doing those in a separate PR, open 2 issues for that so we could track it :)

**Comment by itoogii** [comment]
Thank you @agamm

**Pull request merged**